code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -628,6 +628,10 @@
 /obj/machinery/porta_turret/centcom_shuttle/setup()
 	return
 
+/obj/machinery/porta_turret/centcom_shuttle/weak

```

Comment:
Think you're missing some vars here. Before you had it as a child of aux_base, which had the faction set to neutral. /centcom_shuttle is set to "turret", which is everything but simple_animals, drones, and other turrets.

Also had these before
```
desc = "A turret built with substandard parts and run down further with age. Still capable of delivering lethal lasers to the odd space carp, but not much else."
icon_state = "standard_lethal"
```